### PR TITLE
fixes mm-831 post timestamps now update

### PR DESCRIPTION
**Pull request by jwilander**
@nickago scrolling doesn't seem to be an issue if we just forceUpdate() the post_info and post_header controls. Feel free to pull locally and test it out. @ralder comments are welcome

### CODE REVIEW

**Comment by ralder** [comment]
it seems it works well

File: web/react/components/post.jsx [hunk 1]
```diff
@@ -80,9 +84,9 @@ module.exports = React.createClass({
                     </div>
                     : null }
                     <div className="post__content">
-                        <PostHeader post={post} sameRoot={this.props.sameRoot} commentCount={commentCount} handleCommentClick={this.handleCommentClick} isLastComment={this.props.isLastComment} />
+                        <PostHeader ref="header" post={post} sameRoot={this.props.sameRoot} commentCount={commentCount} handleCommentClick={this.handleCommentClick} isLastComment={this.props.isLastComment} />
                         <PostBody post={post} sameRoot={this.props.sameRoot} parentPost={parentPost} posts={posts} handleCommentClick={this.handleCommentClick} />
-                        <PostInfo post={post} sameRoot={this.props.sameRoot} commentCount={commentCount} handleCommentClick={this.handleCommentClick} allowReply="true" />
+                        <PostInfo ref="info" post={post} sameRoot={this.props.sameRoot} commentCount={commentCount} handleCommentClick={this.handleCommentClick} allowReply="true" />
```
Comment (ralder) [inline web/react/components/post.jsx]: It's not 'real' PostInfo, mean it's always hidden and 'real' PostInfo is insdide PostHeader component (perhaps it's rudiment from previous versions)

Comment (jwilander) [inline web/react/components/post.jsx]: It's actually shown in desktop view (when the screen is wide enough) and then the one in the PostHeader is used for mobile view (when the screen is narrow enough)

Comment (ralder) [inline web/react/components/post.jsx]: Ah, yes, i missed

**Comment by nickago** [comment]
@jwilander yah, this looks really good. Merge at your convenience

**Pull request merged**